content/en/docs/configuration/acme/_index.md

```diff
@@ -120,6 +128,27 @@ spec:
           class: nginx
 ```
 
+### Reusing an ACME Account
+
+You may want to reuse a single ACME account across multiple clusters. This
+might especially be useful when using EAB. If the `disableAccountKeyGeneration`
+field is set, cert-manager will not create a new ACME account and use the
+existing key specified in `privateKeySecretRef`.
+
+
+```yaml
+apiVersion: cert-manager.io/v1beta1
```

Comment:
I guess we can change that to v1 now?

Reply:
That's a good question. I didn't do that since that `apiVersion` doesn't exist quite yet. But, happy to change it if that's preferred.

Reply:
Since this got added in v1.0.0 I will merge the PR into the docs for that release. (which is in less than a week)

Reply:
```suggestion
apiVersion: cert-manager.io/v1
```